ow_lander/scripts/activity_grind.py

```diff
@@ -7,52 +7,123 @@
 import constants
 import math
 import copy
-from tf.transformations import quaternion_from_euler
 from utils import is_shou_yaw_goal_in_range
-from activity_full_digging_traj import move_to_pre_trench_configuration
 from activity_full_digging_traj import go_to_Z_coordinate, change_joint_value
 
-def plan_cartesian_path_lin(move_arm, length):
+def arg_parsing(req):
+  if req.use_defaults :
+    # Default trenching values
+    x_start = 1.65
+    y_start = 0.0
+    depth = 0.05
+    length = 0.6
+    parallel = True
+    ground_position = constants.DEFAULT_GROUND_HEIGHT
+    delete_prev_traj = False
+
+  else :
+    x_start = req.x
+    y_start = req.y
+    depth = req.depth
+    length = req.length
+    parallel = req.parallel
+    ground_position = req.ground_position
+    delete_prev_traj = req.delete_prev_traj
+
+  return [req.use_defaults, x_start, y_start, depth, length, parallel, ground_position, delete_prev_traj]
+
+
+def plan_cartesian_path(move_group, length, alpha, parallel):
+
+  if parallel==False:
```

Comment:
Small style comment: for boolean variables such as this one, 'parallel' and '! parallel' are more succinct than 'parallel==True' and 'parallel==False'